src/humanize/locale/bn_BD/LC_MESSAGES/humanize.po

```diff
@@ -0,0 +1,291 @@
+# Bengali translations for humanize package.
+# Copyright (C) 2021 THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the humanize package.
+# Wasi Master <arianmollik323@gmail.com>, 2021.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
```

Comment:
This is the name and version of the package.

You can do something like one of the other language files: 

```
src/humanize/locale/ca_ES/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/de_DE/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/es_ES/LC_MESSAGES/humanize.po:"Project-Id-Version: \n"
src/humanize/locale/fa_IR/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/fi_FI/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/fr_FR/LC_MESSAGES/humanize.po:"Project-Id-Version: PROJECT VERSION\n"
src/humanize/locale/id_ID/LC_MESSAGES/humanize.po:"Project-Id-Version: \n"
src/humanize/locale/it_IT/LC_MESSAGES/humanize.po:"Project-Id-Version: \n"
src/humanize/locale/ja_JP/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/ko_KR/LC_MESSAGES/humanize.po:"Project-Id-Version: PROJECT VERSION\n"
src/humanize/locale/nl_NL/LC_MESSAGES/humanize.po:"Project-Id-Version: PROJECT VERSION\n"
src/humanize/locale/pl_PL/LC_MESSAGES/humanize.po:"Project-Id-Version: 0.0.1\n"
src/humanize/locale/pt_BR/LC_MESSAGES/humanize.po:"Project-Id-Version: \n"
src/humanize/locale/pt_PT/LC_MESSAGES/humanize.po:"Project-Id-Version: \n"
src/humanize/locale/ru_RU/LC_MESSAGES/humanize.po:"Project-Id-Version: PROJECT VERSION\n"
src/humanize/locale/sk_SK/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/sl_SI/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/sv_SE/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/tr_TR/LC_MESSAGES/humanize.po:"Project-Id-Version: humanize\n"
src/humanize/locale/uk_UA/LC_MESSAGES/humanize.po:"Project-Id-Version: PROJECT VERSION\n"
src/humanize/locale/vi_VN/LC_MESSAGES/humanize.po:"Project-Id-Version: PROJECT VERSION\n"
src/humanize/locale/zh_CN/LC_MESSAGES/humanize.po:"Project-Id-Version: 1.0\n"
```